### PR TITLE
Rename "--manifest" flag to "--pipelines-file"

### PR DESCRIPTION
**Pull request by gaganhegde**
**What type of PR is this?**
> kind cleanup

**What does does this PR do / why we need it**:
Changes the --manifest flag to the --pipelines-file which is more representative of the action it performs.

**Which issue(s) this PR fixes**:
https://issues.redhat.com/browse/GITOPS-160

co authored by @ishitasequeira

### CODE REVIEW

**Review by wtam2018 — APPROVED** [review]

**Comment by bigkevmcd** [comment]
I'd prefer to have seen use of the `PersistentFlags` and `TraverseChildren` mechanism, rather than duplicating it across _every_ command.

**Comment by bigkevmcd** [comment]
See https://github.com/spf13/cobra#persistent-flags, https://github.com/spf13/cobra#local-flag-on-parent-commands and https://github.com/spf13/cobra#bind-flags-with-config

File: pkg/odo/cli/pipelines/environment/add.go [hunk 1]
```diff
@@ -28,14 +28,14 @@ var (
...
-	*genericclioptions.Context
-}
+// type AddEnvParameters struct {
+// 	envName       string
+// 	output        string
+// 	pipelinesFile string
+// 	cluster       string
+// 	// generic context options common to all commands
+// 	*genericclioptions.Context
+// }
```
Comment (wtam2018) [inline pkg/odo/cli/pipelines/environment/add.go:39]: AddEnvParameters should stay in add.go and should not be moved to environment.go.

File: pkg/odo/cli/pipelines/environment/add.go [hunk 2]
```diff
@@ -81,12 +80,10 @@ func NewCmdAddEnv(name, fullName string) *cobra.Command {
 		Example: fmt.Sprintf(addEnvExample, fullName),
 		Run: func(cmd *cobra.Command, args []string) {
 			genericclioptions.GenericRun(o, cmd, args)
+			fmt.Println("These are the pipelines and env-name", o.pipelinesFile, o.envName)
```
Comment (wtam2018) [inline pkg/odo/cli/pipelines/environment/add.go:83]: Please delete.

File: pkg/odo/cli/pipelines/environment/add.go [hunk 3]
```diff
@@ -81,12 +80,10 @@ func NewCmdAddEnv(name, fullName string) *cobra.Command {
...
 		},
 	}
 
-	addEnvCmd.Flags().StringVar(&o.envName, "env-name", "", "name of the environment/namespace")
-	addEnvCmd.MarkFlagRequired("env-name")
-	addEnvCmd.Flags().StringVar(&o.manifest, "manifest", "pipelines.yaml", "path to manifest file")
```
Comment (wtam2018) [inline pkg/odo/cli/pipelines/environment/add.go:89]: Actually, environment command has only one child (add) command).   I would not worry about using Persistent Flag here.   I would revert changes to environment comment.

File: pkg/odo/cli/pipelines/service/service.go [hunk 4]
```diff
@@ -24,8 +41,19 @@ func NewCmd(name, fullName string) *cobra.Command {
...
+	// cmd.PersistentFlags().StringVar(&o.serviceName, "service-name", "", "the name of the service to be added")
+	// cmd.PersistentFlags().StringVar(&o.envName, "env-name", "", "the name of the environment where the service will be added")
+	// cmd.PersistentFlags().StringVar(&o.imageRepo, "image-repo", "", "used to push built images")
+	// cmd.PersistentFlags().StringVar(&o.internalRegistryHostname, "internal-registry-hostname", "image-registry.openshift-image-registry.svc:5000", "internal image registry hostname")
+	// cmd.PersistentFlags().StringVar(&o.pipelinesFilePath, "pipelines-file", "pipelines.yaml", "path to pipelines file")
+	// cmd.Flags().AddFlagSet(addCmd.Flags())
+	// // required flags
+	// _ = cmd.MarkFlagRequired("service-name")
+	// _ = cmd.MarkFlagRequired("app-name")
+	// _ = cmd.MarkFlagRequired("env-name")
```
Comment (wtam2018) [inline pkg/odo/cli/pipelines/service/service.go:56]: Like environment command, service ocmmand only has one child.  Let's revert these changes.

**Review by wtam2018 — CHANGES_REQUESTED** [review]
Need to rebase and PR to `pipelines_feature`

**Pull request closed without merging**